client/src/java/com/zimbra/client/ZMailbox.java

```diff
@@ -3434,9 +3434,9 @@ public void postRESTResource(String relativePath, InputStream is, boolean closeI
             clientBuilder.setRetryHandler(new InputStreamRequestHttpRetryHandler());
             HttpClient client = clientBuilder.build();
             int statusCode;
-      
+
             MultipartEntityBuilder builder = MultipartEntityBuilder.create();
-            builder.addBinaryBody("upfile", is, ContentType.create(contentType != null ? contentType: "application/octet-stream"), null);
+            builder.addBinaryBody("upfile", is, ContentType.DEFAULT_BINARY, "file");
```

Comment:
Please remove this contentType input param or update the doc that this param wont be used.
* @param contentType optional content-type header value (defaults to "application/octect-stream")

Reply:
Will do